cfpb/consumerfinance.gov
# Switch to design system tootips in tccp

Now that tooltips are available in the design system, we should use them instead of our own custom implementation in tccp. This does that.

Things to note:
  - The design-system doesn't `forward` the tooltip sass, so it has to be included specifically. I think this may be an oversight that we should correct in the design system index.scss, but I'll defer to @anselmbradford
  - The data-tooltip needs to match the template id of the tooltip contents, so I put a little counter in the existing tooltip macro. This works more or less fine for how we render pages.
 
## Testing
 - Pull and build
 - Load up http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards and click through.
 - Tooltips should look and perform identically to prod.
